crate/structs/src/uuids.rs

```diff
@@ -0,0 +1,79 @@
+use std::{fmt::Display, ops::Deref};
+
+use uuid::Uuid;
+
+use crate::{encrypted_entries::UUID_LENGTH, error::result::StructsResult, StructsError};
+
+#[derive(Debug)]
+pub struct Uuids {
+    pub uuids: Vec<Uuid>,
+}
+
+impl Deref for Uuids {
+    type Target = Vec<Uuid>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.uuids
+    }
+}
+
+impl Display for Uuids {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        for uuid in &self.uuids {
+            writeln!(f, "UUID: {uuid}")?;
+        }
+        Ok(())
+    }
+}
```

Comment:
We already talked about this, but encapsulating data in ad-hoc wrappers for the same of automatic `Display` logging really bugs me. Is there really no way to use an automatic `Debug` mode instead? I agree it wouldn't be as pretty, but the overhead of these wrappers is really big (79LoC here, 134LoC for the encrypted entries... and it is contaminating).

Reply:
Yes we already talked about this. I didn't find a better way to have neat logs where data are not raw u8 arrays. IMHO, Debug default trait is insufficient if we plan to debug code later. Debugging could be done by someone else. (The Display trait impl' themselves do not take 79LoC and 134LoC)

Reply:
the `Display` implementation isn't, but if you don't need it, then the wrappers are not used and the whole `uuids.rs` and `encrypted_entries.rs` files and surely others become useless.